### PR TITLE
Created homework_3

### PR DESCRIPTION
**Pull request by Doctor-Ledenec**

### CODE REVIEW

File: src/homework_3/Student.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+
+    @Override
+    public String toString() {
+        return String.format("Name = %s \nfileName = %s", this.name, this.fileName);
+    } 
+
+
+    public int checkFileExtension() throws StudentFileException {
```
Comment (SofiaVynnytska12) [inline src/homework_3/Student.java:48]: and what if file is null?

File: src/homework_3/Student.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        int point = 0;
+        File file = new File(this.fileName);
+
+        if (file.length() == 0) {
+            point = -1;
+            throw new StudentFileException("Student file is empty, point = " + point);
+        } else {
+            if (this.fileName.contains(".java")) {
```
Comment (SofiaVynnytska12) [inline src/homework_3/Student.java:56]: better endsWith(...)

**Review by SofiaVynnytska12 — COMMENTED** [review]
Please, resolve merge conflicts

**Comment by Doctor-Ledenec** [comment]
I`ve fixed all of your remerks

**Comment by SofiaVynnytska12** [comment]
I meant that you haven`t predicted the situation like this.fileName = null. It`ll throw an exception, which is not handled. And, please, don`t resolve the conversation on your own, but rather resolve the merge conflicts.

**Pull request closed without merging**